bluesky_kafka/tests/conftest.py

```diff
@@ -169,6 +204,8 @@ def consume_documents_from_kafka_until_first_stop_document(kafka_bootstrap_serve
     ----------
     kafka_bootstrap_servers : pytest fixture
         comma-delimited str of Kafka bootstrap server host:port specified on the pytest command line
+    test_broker_authorization_config: dict
```

Comment:
The doc strings still have the old name.

Reply:
fixed